setup.sh

```diff
@@ -80,6 +80,15 @@ running() {
     [[ $status == ${code} ]]
 }
 
+# Get latitude and longitude 
+LAT="0.0"
+LONG="0.0"
+PYTHON=$(command -v python3 || command -v python)
+if [ -n "${PYTHON}" ]; then
+    LAT=$(curl -s https://freeipapi.com/api/json | "${PYTHON}" -c "import sys, json; print(json.load(sys.stdin)['latitude'])")
+    LONG=$(curl -s https://freeipapi.com/api/json | "${PYTHON}" -c "import sys, json; print(json.load(sys.stdin)['longitude'])")
```

Comment:
Is it worth changing this to make the request only once?

```bash
    IP_RESPONSE=$(curl -s https://freeipapi.com/api/json)
    LAT=$(echo "$IP_RESPONSE" | "${PYTHON}" -c "import sys, json; print(json.load(sys.stdin)['latitude'])")
    LONG=$(echo "$IP_RESPONSE" | "${PYTHON}" -c "import sys, json; print(json.load(sys.stdin)['longitude'])")
```

Reply:
I'm good with that! Need to update weather.sh too.